### PR TITLE
Base calling up implementation

### PR DESCRIPTION
**Pull request by angy113**

### CODE REVIEW

File: IDT.CallingApp/Main/UI/Components/BottomMenu.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// </copyright>
+
+namespace IDT.CallingApp.Main.UI.Components
+{
+    using IDT.Core.Automation.Page.Base;
+    using IDT.Core.Automation.Selenium;
+    using IDT.Core.Automation.Utilities.CommonAction;
+    using OpenQA.Selenium;
+
+    public class BottomMenu : BaseLayout
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Components/BottomMenu.cs]: Add comments

File: IDT.CallingApp/Main/UI/Components/BottomMenu.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        [IDTFindBy(File ="BottomMenuLocators", Locator ="")]
+        private IWebElement BtnHub;
+
+        public BottomMenu ClickHubMenuOption()
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Components/BottomMenu.cs]: add Comments

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using IDT.Core.Automation.Page.Base;
+    using IDT.Core.Automation.Selenium;
+    using IDT.Core.Automation.Utilities.CommonAction;
+    using NUnit.Tests1.Main.UI.Pages;
+    using OpenQA.Selenium;
+
+    public class CountryListLayout : BaseLayout
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: Add comments

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        private const string FileName = "CountryLayoutLocators";
+
+        [IDTFindBy(File = FileName, Locator = "EtSearch")]
+        private IWebElement EtSearch;
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: What does component represent this WebElement?
also adds the right Controls abbreviation. [see](https://github.com/coretech/br-bdd-automation/blob/master/Documentation/Controls.txt)

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+        [IDTFindBy(File = FileName, Locator = "TvCountry")]
+        private IList<IWebElement> TvCountry;
+
+        public CountryListLayout SetSearchValue (string searchText)
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: add comments

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            CommonActions.SendKeys(this.EtSearch, searchText);
+            return this;
+        }
+
+        public VerificationHomeLayout SelectCountry(string countryName)
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: add comments

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+namespace NUnit.Tests1.Main.UI.Pages
+{
+    using IDT.CallingApp.Main.UI.Layouts;
+    using IDT.Core.Automation.Page.Base;
+    using IDT.Core.Automation.Selenium;
+    using IDT.Core.Automation.Utilities.CommonAction;
+    using OpenQA.Selenium;
+
+    public class VerificationHomeLayout : BaseLayout
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: add comments

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        private const string FileName = "VerificationLayoutLocators";
+
+        [IDTFindBy(File = "VerificationLayoutLocators", Locator = "EtPhoneNumber")]
+        private IWebElement EtPhoneNumber;
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: What does component represent this WebElement?
also adds the right Controls abbreviation. [see](https://github.com/coretech/br-bdd-automation/blob/master/Documentation/Controls.txt)

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+        [IDTFindBy(File = "VerificationLayoutLocators", Locator = "BtnVerifyNumber")]
+        private IWebElement BtnVerifyNumber;
+
+        [IDTFindBy(File = "VerificationLayoutLocators", Locator = "BtnCountryCode")]
+        private IWebElement BtnCountryCode;
+
+        public VerificationHomeLayout SetPhoneNumber(string phoneNumber)
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: Add comments

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            CommonActions.SendKeys(this.EtPhoneNumber, phoneNumber);
+            return this;
+        }
+
+        public void ClickOnVerifyNumberButton()
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: add comments

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            CommonActions.ClickElement(this.BtnVerifyNumber);
+        }
+
+        public CountryListLayout DisplayCountyList()
```
Comment (mabalderramav) [inline IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: add Comments

File: IDT.CallingApp/Tests/UI/Features/Hub.feature [hunk 12]
```diff
@@ -0,0 +1,8 @@
+﻿Feature: Hub
+	In order to avoid silly mistakes
+	As a math idiot
+	I want to be told the sum of two numbers
+
+@mytag
```
Comment (mabalderramav) [inline IDT.CallingApp/Tests/UI/Features/Hub.feature]: Rename this tag or remove if it doesn't use.

File: IDT.CallingApp/Tests/UI/Features/Hub.feature [hunk 13]
```diff
@@ -0,0 +1,8 @@
+﻿Feature: Hub
+	In order to avoid silly mistakes
+	As a math idiot
+	I want to be told the sum of two numbers
```
Comment (mabalderramav) [inline IDT.CallingApp/Tests/UI/Features/Hub.feature]: Improve the description of feature.

File: IDT.CallingApp/Tests/UI/StepsDefinitions/HubSteps.cs [hunk 14]
```diff
@@ -0,0 +1,20 @@
+﻿namespace IDT.CallingApp.Tests.UI.StepsDefinitions
```
Comment (mabalderramav) [inline IDT.CallingApp/Tests/UI/StepsDefinitions/HubSteps.cs]: Please add the header comments in the class

File: IDT.CallingApp/Tests/UI/Features/Hub.feature [hunk 15]
```diff
@@ -0,0 +1,8 @@
...
+
+@mytag
+Scenario: Add two numbers
+	Given I go tu hub page
```
Comment (wMercadoJala) [inline IDT.CallingApp/Tests/UI/Features/Hub.feature]: Is it correct this step ? 
checking the Steps definitions it does not go to Hub page

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs (hunk 4 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: It is an EditText in mobile controls sames as label in web controls

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs (hunk 8 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: It is an EditText in mobile controls sames as textbox in web controls

File: IDT.CallingApp/Main/UI/Components/BottomMenu.cs (hunk 1 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Components/BottomMenu.cs]: Done

File: IDT.CallingApp/Main/UI/Components/BottomMenu.cs (hunk 2 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Components/BottomMenu.cs]: Done

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs (hunk 5 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: Done

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs (hunk 6 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: Done

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs (hunk 7 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: Done

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs (hunk 9 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: Done

File: IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs (hunk 11 above)
Comment (angy113) [reply IDT.CallingApp/Main/UI/Layouts/VerificationHomeLayout.cs]: Done

File: IDT.CallingApp/Tests/UI/Features/Hub.feature (hunk 13 above)
Comment (angy113) [reply IDT.CallingApp/Tests/UI/Features/Hub.feature]: Done

File: IDT.CallingApp/Tests/UI/StepsDefinitions/HubSteps.cs (hunk 14 above)
Comment (angy113) [reply IDT.CallingApp/Tests/UI/StepsDefinitions/HubSteps.cs]: Done

File: IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs (hunk 4 above)
Comment (mabalderramav) [reply IDT.CallingApp/Main/UI/Layouts/CountryListLayout.cs]: haaaa realy.

**Pull request merged**